Gemfile

```diff
@@ -7,11 +7,11 @@ gem 'sinatra-flash', '~> 0.3.0'
 gem 'i18n', '0.6.0'
 gem 'sinatra', '~> 1.4.2'
 gem 'sinatra-contrib', '~> 1.4.0'
-gem 'dm-core',          :git => 'git://github.com/datamapper/dm-core.git', :branch => 'release-1.2'
+gem 'dm-core',          '1.2.1'
 gem 'dm-migrations',    '1.2.0'
 gem 'dm-timestamps',    '1.2.0'
 gem 'dm-validations',   '1.2.0'
-gem 'dm-types',         '1.2.0'
+gem 'dm-types',         '1.2.2'
```

Comment:
We need to update json in order to support Ruby 2.2. But dm-types@1.2.0's dependency was `json (~> 1.5.4)`. So I also update dm-types to 1.2.2 which supports newer version of json.